### ISSUE TITLE
Replace inaccessible a tags with buttons

### ISSUE THREAD

**Issue by mulholo** [matildaferrand/matildaferrand.github.io#3]
<a /> tags should be links, not buttons. For accessibility reasons, this line and any others like it should use <button /> tags.

https://github.com/matildaferrand/matildaferrand.github.io/blob/master/index.html#L40
